src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -2683,6 +2683,31 @@ public void Fo()
 }");
         }
 
+        [WpfFact]
+        public void TestSwitchExpression()
```

Comment:
Can we add a few more input scenario tests for this? Also avoid using "Foo" for names.

First that comes to mind:

```
(c) switch$$
```

```
public int M(int c) => (c+5) switch$
```

Reply:
Both tests are added